packages/citation-js-utils/package.json

```diff
@@ -4,7 +4,6 @@
   "description": "Utilities for citation-js.",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
-  "unpkg": "dist/index.umd.min.js",
```

Comment:
These were copy-paste hangovers and were never built (for `citation-js-utils` and `jtex`)